admin.py

```diff
@@ -0,0 +1,45 @@
+class Admin:
+    def __init__(self):
+        self.drinks = {}
+        self.dict_employees = {}
```

Comment:
Does the administrator not have its own attributes?
Name, date of birth, etc.

Reply:
Apart from the name, it makes no sense to use the date of birth or anything else